crates/sui-source-validation-service/src/lib.rs

```diff
@@ -40,8 +40,15 @@ pub struct Packages {
     pub paths: Vec<String>,
 }
 
-/// Map (package address, module name) tuples to verified source paths.
-type SourceLookup = BTreeMap<(AccountAddress, Symbol), PathBuf>;
+#[derive(Debug)]
+pub struct SourceInfo {
+    pub path: PathBuf,
+    // Is Some when content is hydrated from disk.
+    pub source: Option<String>,
```

Comment:
We can hydrate on first request / read at a later point. Right now we do it on server start.